documentation/course/E1.md

```diff
@@ -269,6 +276,8 @@ classDiagram
     class Stop {
         long id [1] #lcub;id, readonly#rcub;
         String name [1] #lcub;readonly#rcub;
+        BigDecimal latitude [1] #lcub;readonly#rcub;
+        BigDecimal longitude [1] #lcub;readonly#rcub;
```

Comment:
**nit:** Very minor, but I think that there's little benefit to using `BigDecimal` here instead of `double`. It was obviously required for money, but if we misplace the stop a few centimeters in any direction it isn't a problem.

Reply:
https://stackoverflow.com/a/1196429

Fair, it is not critical in any sense. It reduces the size of a row by few bytes. We don't need the resolution of floats, but also we don't plan to do any operations on geospatial data. Its purely for administrators convenience. 

We'll go with floats then.